lib/travis/api/v3/services/cron/create.rb

```diff
@@ -0,0 +1,24 @@
+module Travis::API::V3
+  class Services::Cron::Create < Service
+    result_type :cron
+    params :interval, :disable_by_build
+
+    def run!
+      raise LoginRequired unless access_control.logged_in? or access_control.full_access?
+      raise NotFound      unless repository = find(:repository)
+      raise NotFound      unless branch = find(:branch, repository)
+      raise ArgumentError, 'interval must be "daily", "weekly" or "monthly"' unless ["daily", "weekly", "monthly"].include?(params["interval"])
```

Comment:
This should probably be a `WrongParameter` exception or anything else that inherits from `V3::Error`, otherwise this will show up in our exception tracker.